server/node.go

```diff
@@ -107,7 +108,7 @@ func allocateStoreIDs(nodeID proto.NodeID, inc int64, db *client.KV) (proto.Stor
 //
 // Returns a KV client for unittest purposes. Caller should close
 // the returned client.
-func BootstrapCluster(clusterID string, eng engine.Engine) (*client.KV, error) {
+func BootstrapCluster(clusterID string, eng engine.Engine, ctx *Context) (*client.KV, error) {
```

Comment:
It might make sense to simply drop the context here. The reason is that the linearizable flag can very safely be set to "false" here. Bootstrapping the cluster happens just once and is a short-lived affair. There's no reason to insist on linearizable external consistency because no distributed transactions are run, and certainly none with causal, back-channel communication!